### PR TITLE
TiQ tag integration fixes

### PR DESCRIPTION
**Pull request by tamayok**

### CODE REVIEW

File: appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerTracker.kt [hunk 1]
```diff
@@ -1,24 +1,29 @@
 package com.tealium.remotecommands.appsflyer
 
+import android.app.Activity
 import android.app.Application
 import android.content.Context
```
Comment (jameskeith) [inline appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerTracker.kt:5]: Unused import.

File: appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt [hunk 2]
```diff
@@ -194,32 +187,32 @@ open class AppsFlyerRemoteCommand : RemoteCommand {
...
-        val host: String? = payload.optString(Host.HOST)
+    private fun setHost(payload: JSONObject) {
+        val host: String = payload.optString(Host.HOST)
         val hostPrefix: String = payload.optString(Host.HOST_PREFIX)
 
-        host?.let { hostName ->
+        if(host.isNotEmpty()) {
             if (hostPrefix.isNotEmpty()) {
-                tracker.setHost(hostName, hostPrefix)
+                tracker.setHost(hostPrefix, hostPrefix)
```
Comment (jameskeith) [inline appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt:196]: this change doesn't look correct - should we be passing the `hostPrefix` in twice?

File: appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt [hunk 3]
```diff
@@ -194,32 +187,32 @@ open class AppsFlyerRemoteCommand : RemoteCommand {
...
     }
 
-    fun setHost(payload: JSONObject) {
-        val host: String? = payload.optString(Host.HOST)
+    private fun setHost(payload: JSONObject) {
+        val host: String = payload.optString(Host.HOST)
         val hostPrefix: String = payload.optString(Host.HOST_PREFIX)
 
-        host?.let { hostName ->
+        if(host.isNotEmpty()) {
```
Comment (jameskeith) [inline appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt:194]: really minor, but can we get a space between the `if` and opening `(`

File: appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt (hunk 2 above)
Comment (tamayok) [reply appsflyer/src/main/java/com/tealium/remotecommands/appsflyer/AppsFlyerRemoteCommand.kt:196]: Ah, good catch! Fixed.

**Pull request merged**